code/controllers/UserController.php

```diff
@@ -17,8 +17,9 @@ public function index()
     {
         if (!Member::currentUserID()) {
             return $this->redirect('/Security/login');
-        }
-        return $this->renderWith(array('UserController', 'Page'));
+        } else if (UserInvitation::create()->canCreate(Member::CurrentUser()) == false){
```

Comment:
I've run into a problem. Because adding the permission check to `init` runs it on every route action. That means that when the user follows the email link they encounter the permission check which fails because they aren't logged in. Thoughts?